components/datetime/tests/simple_test.rs

```diff
@@ -25,19 +23,19 @@ const EXPECTED_DATETIME: &[&str] = &[
     "शुक्रवार, 22 दिसंबर 2023, 9:22 pm",
     "December 22, 2023, 9:22:53 PM",
     "22 décembre 2023, 21:22:53",
-    "2023年12月22日 21:22:53",
```

Comment:
I'm inclined to pull this behavior change out to a separate PR because it isn't really caused by the new dynamic field sets. It is just exposed by them.

Reply:
I tried splitting it, but it was more work than I thought was needed, because I would need to re-write this test with the old neo skeleton API only to immediately replace it with the new dynamic field set API.